pkgs/peakrdl.nix

```diff
@@ -0,0 +1,150 @@
+# Copyright lowRISC Contributors.
+# SPDX-License-Identifier: MIT
+{pkgs}: let
+  inherit (pkgs) python3Packages fetchFromGitHub lib;
+
+  systemrdl-compiler = python3Packages.buildPythonPackage {
+    name = "systemrdl-compiler";
+    version = "1.27.3";
+    src = fetchFromGitHub {
+      owner = "SystemRDL";
+      repo = "systemrdl-compiler";
+      rev = "191f4dd9bc093920720ddd47e8a15be97e6c5a25";
+      hash = "sha256-HEDN2VeYEA5ZJFCzk4ifhr8VVd8G+NVa/y/kK4ux2BY=";
+    };
+    propagatedBuildInputs = with python3Packages; [
+      antlr4-python3-runtime
+      colorama
+    ];
+    enableParallelBuilding = true;
+    doCheck = false;
+    meta = {
+      description = "SystemRDL 2.0 language compiler front-end ";
+      homepage = "https://systemrdl-compiler.readthedocs.io";
+      license = lib.licenses.mit;
+    };
+  };
+
+  peakRdlPackage = {
+    name,
+    version,
+    rev,
+    hash,
+    description,
+    additionalInputs ? [],
+    mainProgram ? null,
+  }:
+    python3Packages.buildPythonPackage {
+      inherit name version;
+      src = fetchFromGitHub {
+        owner = "SystemRDL";
+        repo = name;
+        inherit rev hash;
+      };
+      propagatedBuildInputs = with python3Packages;
+        [
+          setuptools
+          setuptools-scm
+          systemrdl-compiler
+        ]
+        ++ additionalInputs;
+      format = "pyproject";
+      meta = {
+        inherit description mainProgram;
+        homepage = "https://github.com/SystemRDL/${name}";
+        license = lib.licenses.gpl3;
+      };
+    };
+in rec {
```

Comment:
It looks like this `in rec {` can be removed, and have `in peakrdl` added to the very end?